code-postprocessing/cocopp/rungenericmany.py

```diff
@@ -596,6 +596,8 @@ def main(argv=None):
                         latex_commands_file)
             plt.rcdefaults()
             print_done()
+        if prepare_figures or prepare_tables or prepare_RLDistr or prepare_scatter:
```

Comment:
Why the condition? It seems to be inconsistent with line 292, where this output is printed unconditionally. Isn't it also bug prone to condition the output on a list which could easily extend when the code evolves and hence break this line?

Reply:
I also would think that the output should always appear.

Reply:
I agree completely.

A similar line was already there in rungeneric1.py, so I chose to follow suit in rungenericmany.py. There may have been reasons to make it conditional like this. It would have taken time to investigate. For now, I prefer to address the actual subject of the issue. I try to avoid turning the issue into an excuse to question why the code is there the way it is. There will be a rewrite anyway. All code will go through review before making it to the new code base. Made sense to me to minimize effort spent on code that will likely not be preserved.

That being said, since we already did go the distance for this specific bit and we all agree on what is better, I will remove the condition.